### DISCUSSION TITLE
Can ping container from NPM console, but can't use container name for a Proxy Host

### DISCUSSION THREAD

**Discussion by Iceman248** [NginxProxyManager/nginx-proxy-manager#3725] [Q&A]
I put both NPM and a container on the same network, then could ping it, but it still just gives 502 Bad Gateway when trying to use it. Is there something else that I am not understanding about how this is supposed to work>

**Comment by essys** [comment]
Please share your docker cmd or docker compose files.
I had a similar issue in the past and in the end it was an issue with the docker network and the syntax in my compose files.

**Reply by Iceman248** [reply]
Thanks, although I don't have either of those for either of the 2 setups I was testing it with. One uses the community templates from unRAID and the other was creating as a Portainer Container.
